screens/AccountClosureForm.js

```diff
@@ -0,0 +1,211 @@
+import InputText from '../components/InputText'
+import { isRequired } from '../validators'
+
+class AccountClosureForm extends React.Component {
+  static navigationOptions = {
+    header: null
+  }
+  static contextType = LocaleContext
+
+  constructor(props, context) {
+    super(props, context)
+
+    context.localize({
+      en: {
+        Title: 'Account Closure',
+        staff: 'Staff',
+        cash: 'Cash',
+        card: 'Credit Card',
+      	next: 'Next',
+      	status: 'Status',
+      	startingCash: 'Starting Cash',
+      	totalCashInRegister: 'Total Cash In Register',
+      	remark: 'Remark',
+      	totalCardTransactions: 'Total Credit Card Transactions'      	
+      },
+      zh: {
+        Title: 'Account Closure-CH',
+        staff: '員工',
+        cash: 'Cash-CH',
+        card: 'Credit Card-CH',
+        next: 'Next-CH',
+        status: 'Status-CH',
+        startingCash: 'Starting Cash-CH',
+        totalCashInRegister: 'Total Cash In Register-CH',
+        remark: 'Remark-CH',
+        totalCardTransactions: 'Total Credit Card Transactions-CH'
+      }
+    })
+  }
+
+  render() {
+    const { t } = this.context
+    const { mostrecentShift, handleSubmit } = this.props
+
+		if(mostrecentShift !== null){
+    return (
+      <DismissKeyboard>
+      	<ScrollView>          
+
+					{/* Cash */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('cash')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop, {marginBottom: 8}]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>
+              	<View style={[styles.fieldContainer]}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	{t('startingCash')}
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;{mostrecentShift.open.balance}</Text>
+                	</View>
+              	</View>     	         
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	{t('totalCashInRegister')}
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
+              				name="cashclosingBalance"
+              				component={InputText}
+              				validate={isRequired}
+              				placeholder={"0"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+
+              	<View style={styles.fieldContainer}>
```

Comment:
should also include total cash transactions here.

Reply:
??